core/sets.gms

```diff
@@ -1766,12 +1773,7 @@ entySeSyn(all_enty)   "synfuel secondary energy types"
   segasyn    "secondary energy gas from H2"
 /
 
-entySeFos(all_enty) "secondary energy types from fossil primary energy"
-/
-        seliqfos     "secondary energy liquids from fossil primary energy"
-        sesofos      "secondary energy solids from fossil primary energy"
-        segafos      "secondary energy gas from fossil primary energy"
-/
+entySeNonFos(all_enty)   "secondary energy types from non-fossil primary energy"
```

Comment:
I find that name misleading: It suggests that the set contains `seel`, `sehe` and `seh2`.